todoman/model.py

```diff
@@ -587,6 +589,12 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             params.extend(lists)
         if urgent:
             extra_where.append('AND priority = 9')
+        elif priority:
```

Comment:
I think this line has to be `elif priority is not None`, because if `priority` is `0`, this branch is supposed to be used too